Doppler.HtmlEditorApi.Test/GetCampaignTest.cs

```diff
@@ -121,12 +123,15 @@ public async Task GET_campaign_should_accept_right_tokens_and_return_404_when_DB
         public async Task GET_campaign_should_accept_right_tokens_and_return_content_as_it_is_in_DB(string url, string token, string expectedAccountName, int expectedIdCampaign)
         {
             // Arrange
-            var dbContent = "{ \"hi\": \"this is an Unlayer's design!\" }";
+            Model.ContentModel emptyContentModel = new Model.ContentModel()
```

Comment:
Why are you using empty as a prefix?
```suggestion
            Model.ContentModel contentModel = new Model.ContentModel()
```

Reply:
Its ok now!